projects/core/src/lib/project/project.service.ts

```diff
@@ -1619,3 +1663,11 @@ export class ProjectService {
         this.addLayer(layer);
     }
 }
+
+/**
+ * Used as filter argument for T | undefined
+ */
+// eslint-disable-next-line prefer-arrow/prefer-arrow-functions
+function isDefined<T>(arg: T | null | undefined): arg is T {
```

Comment:
Haben wir das nicht schon irgendwo im Projekt mal definiert, in einer utility-klasse?